packages/atomic/tailwind.config.js

```diff
@@ -2,53 +2,54 @@ const isDevWatch = process.argv.indexOf('--dev') > -1;
 
 module.exports = {
   purge: {
-    content: [
-      './src/**/*.tsx', './src/**/*.css'
-    ],
-    enabled: !isDevWatch
+    content: ['./src/**/*.tsx', './src/**/*.css'],
+    enabled: !isDevWatch,
   },
   darkMode: false, // or 'media' or 'class'
   theme: {
     extend: {
       colors: {
-        "primary": "var(--atomic-primary)",
-        "primary-variant": "var(--atomic-primary-variant)",
-        "on-primary": "var(--atomic-on-primary)",
-        "secondary": "var(--atomic-secondary)",
-        "secondary-variant": "var(--atomic-secondary-variant)",
-        "on-secondary": "var(--atomic-on-secondary)",
-        "background": "var(--atomic-background)",
-        "background-variant": "var(--atomic-background-variant)",
-        "on-background": "var(--atomic-on-background)",
-        "on-background-variant": "var(--atomic-on-background-variant)",
-        "divider": "var(--atomic-divider)",
-        "error": "var(--atomic-error)",
-        "on-error": "var(--atomic-on-error)",
-        "visited": "var(--atomic-visited)",
+        // Primary
+        primary: 'var(--atomic-primary)',
+        'primary-light': 'var(--atomic-primary-light)',
+        'primary-dark': 'var(--atomic-primary-dark)',
+        'on-primary': 'var(--atomic-on-primary)',
+        // Secondary
+        secondary: 'var(--atomic-secondary)',
+        'secondary-light': 'var(--atomic-secondary-light)',
+        'secondary-dark': 'var(--atomic-secondary-dark)',
+        'on-secondary': 'var(--atomic-on-secondary)',
+        // Neutral
+        neutral: 'var(--atomic-neutral)',
+        'neutral-light': 'var(--atomic-neutral-light)',
+        'neutral-dark': 'var(--atomic-neutral-dark)',
+        // Semantic
+        background: 'var(--atomic-background)',
+        'on-background': 'var(--atomic-on-background)',
+        success: 'var(--atomic-success)',
+        error: 'var(--atomic-error)',
       },
       borderRadius: {
```

Comment:
From the design, only 2 radius are necessary, normal (4px) and large (16px)

Reply:
When implementing, we always have to be conscious as to what variable we expose and reuse those values